lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -878,17 +884,14 @@ def buildUI(self):
                     self.createSection(sectionName, attrsToAdd, collapse)
 
     def suppressArrayAttribute(self):
-        # Suppress all array attributes except UsdGeom.Tokens.xformOpOrder
+        # Suppress all array attributes.
         if not self.showArrayAttributes:
             for attrName in self.attrS.attributeNames:
                 if self.isArrayAttribute(attrName):
                     self.suppress(attrName)
 
     def isArrayAttribute(self, attrName):
-        # Note: UsdGeom.Tokens.xformOpOrder is a exception,
-        #       we are not considering this attribute as an array.
-        if self.attrS.attributeType(attrName) == ufe.Attribute.kGeneric and \
-           attrName != UsdGeom.Tokens.xformOpOrder:
```

Comment:
We no longer want xformOpOrder to be an exception for the "Show Array Attributes" toggle.